spec/lib/vet360/models/base_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Vet360::Models::Base do
+  describe '#default_attrs_present?' do
+    it "returns true if the model instance's has all of the DEFAULT_ATTRS" do
```

Comment:
I think I understand the intent of this test, but to anyone who doesn't know exactly what's going on in the factories, it's not obvious why `vet360_message.default_attrs_present?` should fail. It might be more intuitive to `nil` out a default attr on the previous Email model to demonstrate this functionality.

Reply:
It is a predicate check that is expected to return `true` or `false`.